man/systemd.special.xml

```diff
@@ -1018,7 +1018,14 @@
             in. systemd automatically adds dependencies of type
             <varname>After=</varname> for this target unit to all SysV
             init script service units with an LSB header referring to
-            the <literal>$time</literal> facility. </para>
+            the <literal>$time</literal> facility, and also to all timer
+            units with at least one <varname>OnCalendar=</varname>
+            directive. </para>
+
+            <para>Unless
+            <citerefentry><refentrytitle>systemd-time-wait-sync.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+            is enabled, this target might get reached way before the
```

Comment:
'get reached way'? Maybe, 'get reached' is enough. No?

Reply:
Agreed, the language is too informal for a manpage. I force-pushed an update that removes "way" and changes "time" to "clock".